src/GitHub.VisualStudio/UI/Views/PullRequestDetailView.xaml

```diff
@@ -448,7 +448,8 @@
                                                 <TextBlock.Style>
                                                     <Style TargetType="TextBlock">
                                                         <Style.Triggers>
-                                                            <DataTrigger Binding="{Binding ChangeType}" Value="Removed">
+                                                            <DataTrigger Binding="{Binding Status}" Value="Removed">
+                                                                <Setter Property="Foreground" Value="Red"/>
```

Comment:
Colors shouldn't be hardcoded, as the UI needs to adapt to theme changes. We have different files defining the colors for each theme under the `Styles` folder, any control that needs theme-aware colors should reference resources from there.